hotel_restaurant/models/hotel_restaurant.py

```diff
@@ -161,7 +161,8 @@ def table_done(self):
     _name = "hotel.restaurant.reservation"
     _description = "Includes Hotel Restaurant Reservation"
 
-    reservation_id = fields.Char('Reservation No', size=64, required=True, default=lambda obj: obj.env['ir.sequence'].get('hotel.restaurant.reservation'))
+    #reservation_id = fields.Char('Reservation No', size=64, required=True, default=lambda obj: obj.env['ir.sequence'].get('hotel.restaurant.reservation'))
+    reservation_id = fields.Char('Reservation No', size=64, required=True)
     room_no = fields.Many2one('hotel.room', string='Room No', size=64)
```

Comment:
A numerical sequence for documents should create the number on the recording act in the database.
It's not what is happening. If I create a reserve is assigned a number and when I record changes the number.
If I discard the creation of reserves, the numerical sequence always advances a number.
In standar method this problem does not occur